layouts/shortcodes/latestBintrayVersion.html

```diff
@@ -0,0 +1 @@
+{{ $url := .Get "url" }}{{ $versionInfo := getJSON $url }}{{ $versionInfo.name }}
```

Comment:
I like what you've done here. I wonder if it would be improved by including the group ID and artifact ID as well, thereby making the shortcode specific to Gradle and ensuring that changes to the build artifacts are reflected automatically. For example:

Markdown:
compile '{{% latestGradleConfiguration url="https://api.bintray.com/packages/nasaworldwind/maven/WorldWindAndroid" %}}'

Shortcode:
{{ $url := .Get "url" }}{{ $versionInfo := getJSON $url }}{{ $versionInfo.system_ids[0] }}":"{{ $versionInfo.latest_version }}"@aar"

This shortcode may be incorrect, but I think you'll get the idea. The output is the string:
gov.nasa.worldwind.android:worldwind:0.6.0@aar